### DISCUSSION TITLE
phantom and coinbase

### DISCUSSION THREAD

**Discussion by ryuk8000** [WalletConnect/walletconnect-monorepo#2470] [Developer Support]
https://lab.web3modal.com/v1ManagedReact
i am connecting my wallet through this link but phantom and coinbase is not shown as installed wallets. But when i go to this link thorugh browser provided by phantom, I can see phantom in other wallets. What could be the issue. Should i install only one wallet out of metamask , coinbase or phantom ?

**Comment by ryuk8000** [comment]
i am connecting wallet through mobile . so this query is related when connecting through mobile

**Reply by finessevanes** [reply]
Consider using this link instead: https://lab.web3modal.com/with-sign-api/react. I received a 404 error with the previous link, and that's version 1, which will be deprecated in less than two weeks.
